src/v/cluster/scheduling/leader_balancer_strategy.h

```diff
@@ -60,6 +60,8 @@ class leader_balancer_strategy {
     virtual std::optional<reassignment>
     find_movement(const absl::flat_hash_set<raft::group_id>& skip) const = 0;
 
+    virtual void apply_movement(const reassignment& reassignment) = 0;
+
```

Comment:
i am wondering if we can merge this commit with the one adding implementation for this method as this makes this commit not to build.

Reply:
Good point, will merge this commit with the next that adds the implementation.